azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -0,0 +1,237 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .auth import ConnectionStringAuthentication
+from .protocol.iot_hub_gateway_service_ap_is20190701_preview import (
+    IotHubGatewayServiceAPIs20190701Preview as pl_client,
+)
+from .protocol.models.device import Device
```

Comment:
>from .protocol.models.device  [](start = 0, length = 29)

As in the other file, we should import these like `from .protocol.models import Device` rather than accessing the specific module, as it makes our code less breakable.